docs/source/cli_help/format.rst

```diff
@@ -0,0 +1,31 @@
+.. role:: bash(code)
+
+.. _format:
+
+format
+------
+Note: there is no 'git machete format' command as such; 'format' is just a topic of 'git machete help'.
+
+The format of the definition file should be as follows:
+
+|  develop
```

Comment:
Snippet pls

Reply:
I don't understand your expectations here. 
Do you mean:
 1. file should be in double backticks
 2. file should be a link to file command definitions
 3. Expected representation of the whole tree should be in a code-block, so it's more distinguishable
 ?

Reply:
Sorry! I mean point 3, a code block around the entire tree

Reply:
(same for all other trees across the docs)

Reply:
Based on some following comments I'm assuming (3)